dealers_styles.css

```diff
@@ -0,0 +1,124 @@
+body{
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  color: black;
+  margin: 40px;
+}
+
+.nav {
+  list-style-type: none;
+  padding: 0;
+  display: flex;
+  background-color: rgb(2, 2, 2);
+  align-items: center;
+  width: 100%;
+  height: 40px;
+  }
+
+.nav li {
+  flex: 1;
+  justify-content: center;
+  color: rgb(133, 130, 103);
+ }
+
+ .nav a {
+  font-size: 1.3em;
+  color: #fff;
+  text-decoration: none;
+  padding: 5px;
+  border-radius: 5px;
+}
+.nav a:hover {
+  color: #202b47;
+  background-color: #fff;
+  box-shadow: #d3d3d3;
+}
+
+.nav a {
+  font-size: 1.3em;
+  color: #fff;
+  text-decoration: none;
+  padding: 5px;
+  border-radius: 5px;
+  transition: color 1s, background-color 0.5s;
+}
+
+ a {
+   text-decoration: none;
+ }
+
+ .first_post div{ 
+  display: flex;
+  align-items: center
+}
+
+.first_post p, .post p {
+  font-family:  sans-serif;
+  font-size: 1.5em;
+}
+
+a {
+  color: white;
+  background-color: black;
+}
+
+a:visited {
+  color: white;
+}
+
+.first_post h1 {
+  font-size: 80px;
+  margin-bottom: 16px;
+  text-align: center;
+}
+
+.post img {
+  height: 450px;
+  width: 30%;
+}
+
+#carousel {
+  height: 100vh; text-align: center; 
+ }
+
+#carousel img {
+  height: 100%; width: 100%; 
+ }
+
+#carousel .carousel-image {
```

Comment:
Make a separate line for each css property. Right now there is only one line.
for example: 
position: absolute;
display: block;